variables.styl

```diff
@@ -32,9 +32,4 @@ html
   --transition-speed-quick: 140ms
   --transition-speed-ultra: 33ms
 
-$breakpoint-s = 759px
```

Comment:
I didn't mean to remove the variables. Just rather to use the stylus variables as the single source of truth and set the css variables from them. So in that case we can access both types of variables in derivative code and know that there is only 1 place that the breakpoints are set